_cloudstor/02-generate-cloudstor-password.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Generate App Password
+order: 2
+duration: 5
+---
+
+1. Go to [Cloudstor Dashboard](https://cloudstor.aarnet.edu.au/) and select the university or institution are you based. The login process is via AAF and is exactly the same as login to Nectar Cloud Dashboard.
```

Comment:
AAF - maybe first time using this, we should say 'Australian Access Federation (AAF)', then you're ok to use the abbreviation later